README.md

```diff
@@ -22,6 +22,7 @@ To build your own typechecker:
   Central. Change the name of the package containing the source (currently
   `org.checkerframework.checker.templatefora`) so that it reflects your
   organization's naming standards.
+* Change the `path` specified in the `line 22` of `templatefora-checker-qual/build.gradle` with your actual path.
```

Comment:
I'm very resistant to add an instruction that references a specific line of another file. If that file is ever changed, this instruction will become incorrect.

Unlike the Checker Framework proper, this project does not need to handle multiple qual directories by default. So, we might be able to avoid this issue by specifying `templatefora` directly in the path in `templatefora-checker-qual/build.gradle`, so that the later instructions to modify every file with that string will cause the change we want.

Related, we might want to change the previous instruction to explicitly replace all instances of `org.checkerframework.checker.templatefora` and `org/checkerframework/checker/templatefora`

Reply:
I have reverted the `updated readme` commit.

Reply:
That does not solve the problem. Now the change lacks documentation.

Reply:
Sorry, I forgot to push the recent update.
I have updated the path from `**/org/checkerframework/**/qual/**` to `**/org/checkerframework/checker/templatefora/qual/**` and also updated the readme.